include/ibamr/INSHierarchyIntegrator.h

```diff
@@ -539,9 +539,11 @@ class INSHierarchyIntegrator : public IBTK::HierarchyIntegrator
      */
     SAMRAI::tbox::Pointer<IBTK::CartGridFunction> d_U_init, d_P_init;
     SAMRAI::solv::LocationIndexRobinBcCoefs<NDIM> d_default_bc_coefs;
-    std::vector<SAMRAI::solv::RobinBcCoefStrategy<NDIM>*> d_bc_coefs, d_U_bc_coefs, d_U_star_bc_coefs;
+    std::vector<SAMRAI::solv::RobinBcCoefStrategy<NDIM>*> d_bc_coefs, d_U_bc_coefs;
```

Comment:
Why split?

Reply:
I forgot I split this one. Originally I had written a conversion for that to `std::vector<std::unique_ptr<..>>` but we need the vector of pointers in so many places I just went with what I had working at that point. I'll un-split it.

Reply:
Fixed!